### PR TITLE
Pull request without overwriting antiweb.py

### PR DESCRIPTION
**Pull request by GhostPHX**
... and updates to the documentation. antiweb also installs sphinx and beautifulsoup4 when being installed.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/7enderhead/antiweb/pull/31" target="_blank">ReviewNinja

**Comment by 7enderhead** [comment]
Remove directory 'build'

File: README.md [hunk 1]
```diff
@@ -38,4 +38,20 @@ License: GPL
...
-				Introduced version 0.3.2
\ No newline at end of file
+				Introduced version 0.3.2
+                
+                
+            02.03.2015
+                Enhanced C# support. Now xml comment tags are stripped.
+
+            02.03.2015 - 08.08.2016
+                Many different improvements and bugfixes
```
Comment (7enderhead) [inline README.md:48]: Missing: introduction of version 0.3.3

File: README.md [hunk 2]
```diff
@@ -38,4 +38,20 @@ License: GPL
...
+	
+            08.08.2016
+                Rework of --output path and file name handling
+        
+                Improved antiweb documentation
+    
+            09.08.2016
+                Outsourced installation and getting started (it is no longer available inside the antiweb documentation)
```
Comment (7enderhead) [inline README.md:56]: What about changelog.py? Can the data be synchronized automatically?

File: antiweb.egg-info/SOURCES.txt [hunk 3]
```diff
@@ -8,37 +8,4 @@ antiweb.egg-info/dependency_links.txt
 antiweb.egg-info/requires.txt
 antiweb.egg-info/top_level.txt
 antiweb.egg-info/zip-safe
-tests/block1.c
```
Comment (7enderhead) [inline antiweb.egg-info/SOURCES.txt:11]: Why have these files been removed from here?

File: documentation/build/html/_sources/changelog.txt [hunk 4]
```diff
@@ -0,0 +1,58 @@
+*********
```
Comment (7enderhead) [inline documentation/build/html/_sources/changelog.txt:1]: Either make /README.md use this file or use the contents from there here. Having the same information duplicated is no good.

File: documentation/build/html/_sources/antisphinx.txt [hunk 5]
```diff
@@ -45,7 +45,6 @@ File Layout
 ***********
```
Comment (7enderhead) [inline documentation/build/html/_sources/antisphinx.txt:45]: Please investigate whether we can at least exclude the 'build/html/_sources' directory from version control, or whether the data therein is needed for showing the generated documentation.

**Comment by 7enderhead** [comment]
In general, please always mention authors M. Reitinger, you, Lukas Tanner, Philipp Grandits and me (Christian Eitner), since we all work on antiweb.

File: antiweb.egg-info/SOURCES.txt (hunk 3 above)
Comment (GhostPHX) [inline antiweb.egg-info/SOURCES.txt:11]: antiweb.egg-info/SOURCES.txt is only used for metadata when building a zipped distribution. The recent built distribution deleted those files automatically.

File: documentation/build/html/_sources/antisphinx.txt (hunk 5 above)
Comment (GhostPHX) [inline documentation/build/html/_sources/antisphinx.txt:45]: Sphinx copies the .rst files it processes to /_sources/ as a txt file to make them available as source code on the documentation (through a link in the top right corner)

File: README.md (hunk 2 above)
Comment (GhostPHX) [inline README.md:56]: changelog.py is no longer necessary, it will be removed in the next commit

**Comment by GhostPHX** [comment]
directory 'build' removed

**Pull request merged**